ai21/clients/studio/resources/assistant/studio_assistant.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+from typing import List
+
+from ai21.clients.common.assistant.assistant import Assistant
+from ai21.clients.studio.resources.studio_resource import (
+    AsyncStudioResource,
+    StudioResource,
+)
+from ai21.models.responses.assistant_response import AssistantResponse
+from ai21.types import NotGiven, NOT_GIVEN
+
+
+class StudioAssistant(StudioResource, Assistant):
+    def create(
+        self,
+        name: str,
+        *,
+        description: str | NotGiven = NOT_GIVEN,
+        optimization: str | NotGiven = NOT_GIVEN,
+        avatar: str | NotGiven = NOT_GIVEN,
+        models: List[str] | NotGiven = NOT_GIVEN,
+        tools: List[str] | NotGiven = NOT_GIVEN,
+        tool_resources: dict | NotGiven = NOT_GIVEN,
+        **kwargs,
+    ) -> AssistantResponse:
+        body = self._create_body(
+            name=name,
+            description=description,
+            optimization=optimization,
+            avatar=avatar,
+            models=models,
+            tools=tools,
+            tool_resources=tool_resources,
+            **kwargs,
+        )
+
+        return self._post(path=f"/{self._module_name}", body=body, response_cls=AssistantResponse)
+
+    def get(self, assistant_id: str) -> AssistantResponse:
+        return self._get(path=f"/{self._module_name}/{assistant_id}", response_cls=AssistantResponse)
+
+    def list(self) -> List[AssistantResponse]:
+        response = self._get(path=f"/{self._module_name}")
+        return response["results"]
```

Comment:
Missing `response_cls=List[FileResponse]` no? `response["results"]` is a raw dict